.github/workflows/router-ci.yaml

```diff
@@ -49,6 +49,17 @@ jobs:
         working-directory: ./router
         run: go mod download
 
+      - name: Run go vet
+        run: "go vet ./..."
+        working-directory: ./router
+
+      - name: Run staticcheck linter
+        uses: dominikh/staticcheck-action@v1.3.0
+        with:
+          version: "2023.1.6"
+          install-go: false
+          working-directory: ./router
```

Comment:
Can we run these in router-tests too? Maybe it is more convenient to put both checks into a reusable action, then call it from `graphqlmetrics-ci` once and thrice from `router-ci` (router, router-tests and demo)

Reply:
It is already an action. So you want to put an action into an action? :smile:

Reply:
I have added it to the router tests too.

Reply:
I mean an action for vet + staticcheck. The idea is that we apply the same set of linters across the codebase. If we copy and paste the block across workflows, it might get out of sync.

Reply:
Discussed via messages